content/2_Bronze/Complete_Rec.problems.json

```
@@ -97,6 +97,19 @@
         "kind": "USACO",
         "usacoId": "360"
       }
+    },
+    {
+      "uniqueId": "usaco-692",
+      "name": "Secret Cow Code",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=692",
+      "source": "Silver",
+      "difficulty": "Insane",
+      "isStarred": false,
+      "tags": ["Recursion", "Ad Hoc"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "692"
```

```suggestion
        "kind": "internal"
```
supposed to be like this for internal sols?